bundle/config/resources.go

```diff
@@ -153,6 +193,7 @@ func (r *Resources) ConfigureConfigFilePath() {
 type ConfigResource interface {
 	Exists(ctx context.Context, w *databricks.WorkspaceClient, id string) (bool, error)
 	TerraformResourceName() string
+	Validate(key string) error
```

Comment:
Can we do without the key argument? Feels backward to specify the key of something to a function on itself.